jumpcloud-ADMU/Powershell/Tests/PSScriptAnalyzerSettings.psd1

```diff
@@ -32,15 +32,16 @@
     # Note: if a rule is in both IncludeRules and ExcludeRules, the rule
     # will be excluded.
     ExcludeRules = @('PSUseShouldProcessForStateChangingFunctions',
-                    'PSAvoidUsingWriteHost',
-                    'PSAvoidUsingConvertToSecureStringWithPlainText'
-                    'PSAvoidUsingPlainTextForPassword',
-                    'PSAvoidUsingUsernameAndPasswordParams',
-                    'PSAvoidUsingWMICmdlet',
-                    'PSAvoidUsingInvokeExpression',
-                    'PSUseDeclaredVarsMoreThanAssignments',
-                    'PSUseBOMForUnicodeEncodedFile'
-                    )
+        'PSAvoidUsingWriteHost',
+        'PSAvoidUsingConvertToSecureStringWithPlainText'
+        'PSAvoidUsingPlainTextForPassword',
+        'PSAvoidUsingUsernameAndPasswordParams',
+        'PSAvoidUsingWMICmdlet',
+        'PSAvoidUsingInvokeExpression',
+        'PSUseDeclaredVarsMoreThanAssignments',
+        'PSUseBOMForUnicodeEncodedFile',
+        'PSAvoidGlobalVars'
```

Comment:
I've added `PSAvoidGlobalVairables` so we can print out the test job names while we are running the tests. The logs were getting too difficult to parse out and debug.